examples/pulse_control/1_Bringup_experiments.ipynb

```diff
@@ -58,23 +58,23 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The experiments in this notebook can be run on various devices by setting the parameter `device_name` below. For this notebook, we will focus on Rigetti's Aspen M-3 device."
+    "The experiments in this notebook can be run on various devices by setting the parameter `device_name` below. For this notebook, we will focus on Rigetti's Ankaa-2 device."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
-    "device_name = \"aspen\"\n",
+    "device_name = \"ankaa\"\n",
     "\n",
     "experiment_configuration = {\n",
-    "    \"aspen\": {\n",
-    "        \"device_arn\": Devices.Rigetti.AspenM3,\n",
+    "    \"ankaa\": {\n",
+    "        \"device_arn\": Devices.Rigetti.Ankaa2,\n",
     "        \"qubit\": 4,\n",
```

Comment:
qubit 4 happens to have bad T2 time. Could you use another qubit (eg qubit 3) with higher T2 to make the Ramsey experiment look better?